compiler/modules/front/scan.bal

```diff
@@ -132,6 +139,18 @@ function createFragTokens() returns readonly & FixedToken?[] {
         ft[FRAG_KEYWORD + i] = keywords[i];
     }
     // JBUG int casts needed
+    fillMultipleCharTokens(ft);
```

Comment:
This is ugly: you are separating out of the the three subcases but not the others.

Reply:
I moved it because it was giving a method too large error, what can be done instead?

Reply:
after pulling from the current branch its giving a method too large error for `fillMultipleCharTokens` too

Reply:
Try doing something like toToken in token.bal.